Component/Alert/alert.html

```diff
@@ -0,0 +1,64 @@
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="/Component/Alert/alert.css">
+    <script src="https://kit.fontawesome.com/0ce6b5a07c.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    
+
+    <div class="container">
+        <div class="main-content">
+        
+            <h2 class="heading-1">Alert </h2>
+            <p>Alerts are important information prompts that needs to user attention. Here are few alerts you can make use of.</p>
+            
+            <div class="main-content-display">
+                <li class="alert "> This is a default alert</li>
+                <li class="alert alert-primary"> This is a primary alert</li>
+                <li class="alert alert-success"><i class="fas fa-check-square"></i> This is a success alert</li>
+                <li class="alert alert-error"><i class="fas fa-exclamation-circle"></i> This is an error alert</li>
+                <li class="alert alert-warning"><i class="fas fa-exclamation-triangle"></i> This is a warning alert</li>
+                <li class="alert alert-light">This is Light color alert</li>
+                <li class="alert alert-dark"> This is Dark alert</li>
+            </div>
+            
+            <div class="main-content-example" >
+                
+                <span class="example-comment">Note: The icons used are from fontawesome. You need to add their link to use the icons from the code below<br> <br>
+                
+                //default alert  <br> </span>
+                <<!---->li class="alert "> This is a default alert<<!---->/li> <br> <br>
+
+                <span class="example-comment">//primary alert</span> <br>
+                <<!---->li class="alert alert-primary"> This is a primary alert<<!---->/li> <br><br>
+
+                <span class="example-comment"> //Success alert</span> <br>
+                <<!---->li class="alert alert-success"><<!---->>i class="fas fa-check-square"><<!---->>/i> This is a success alert<<!---->/li> <br><br>
+
+                <span class="example-comment"> //Error alert</span> <br>
+                <<!---->li class="alert alert-error"><<!---->i class="fas fa-exclamation-circle"><<!---->/i> This is an error alert<<!---->/li> <br><br>
+
+                <span class="example-comment"> //Warning alert</span> <br>
+                <<!---->li class="alert alert-warning"><<!---->i class="fas fa-exclamation-triangle"><<!---->/i> This is a warning alert<<!---->/li> <br><br>
+
+                <span class="example-comment"> //Light theme alert</span> <br>
+                <<!---->li class="alert alert-light">This is Light color alert<<!---->/li> <br><br>
+
+                <span class="example-comment"> //dark theme alert</span> <br>
+                <<!---->li class="alert alert-dark"> This is Dark alert<<!---->/li> <br><br>
```

Comment:
Any reason for **<<!---->**li class="alert alert-dark"> **<<!---->/li>** ?
Just Asking...